test/compress/collapse_vars.js

```diff
@@ -1152,60 +1256,3 @@ collapse_vars_short_circuited_conditions: {
         function i6(x) { var a = foo(), b = bar(); if (baz()) return b; else return a; }
     }
 }
-
-collapse_vars_regexp: {
```

Comment:
I was trying to remove my own tests at the end of the file, and forgot I've done so already 🙄 

Tests restored.